src/output.js

```diff
@@ -1,7 +1,7 @@
 const fs = require("fs");
 const path = require("path");
 const { config } = require("./config");
-const batchSize = 10000;
+const batchSize = 10;
```

Comment:
It may be a better idea to remove the batchSize altogether. While testing with larger data with 20+ dynamic case attrs, it seems even 100 rows is too much. 10 works fine with 25 additional attributes for now, but this can fail for a different config. Writing one row at a time is not efficient but I am willing to trade that off for reliability. I will wait for your thoughts.

Reply:
Is it simply a function of the number dynamic attributes?

Reply:
Not exactly. Another scenario is someone updating vocab json to have longer strings as attribute values. It is the total characters it can hold in memory. Smaller batch sizes guarantee success most cases.